runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -135,6 +137,9 @@ pub trait Trait:
         ProposalParameters<Self::BlockNumber, BalanceOf<Self>>,
     >;
 
+    /// Bounty Id type (mirroring bounty::Trait::BountyId)
+    type BountyId: From<u32> + Parameter + Default + Copy;
```

Comment:
Is there a reason why you introduce a new type here? Can't we get away without it like with `blog::PostId` for other proposals?

Reply:
`blog::PostId` is a `pub` alias for `u64`. `BountyId` is a define in `bounty::Trait` as 
```
type BountyId: From<u32> + Parameter + Default + Copy;
```
So 3 alternative comes to my mind:
1. We move `BountyId` to `common` pallet
2. We define something like `pub type BountyId = u64`; in the `bounty` pallet
3. implement the `bounty::Trait` for `proposal_codex::Trait`

Reply:
Can you weight in @shamil-gadelshin ?

Reply:
I recommend option 3 similar to `proposals_engine::Trait` and its `ProposalId`

Reply:
fixed in 57206bdd4604b15286909acb641129139e5f10bd